src/Import/Exports.fs

```diff
@@ -0,0 +1,6 @@
+module FableTemplateJsLib.Exports
+
+open Fable.Core
+
+[<Import("printPairsPadded", from="fable-lib")>]
```

Comment:
Is the `from` parameter to `Import` the JavaScript module providing the function? If so, should we look into replacing this and the `name` property of `package.json` as a part of the template process?

Reply:
Indeed, if we manage to automate the creation of this file, `fable-lib` should be replaced by `name` in package.json. I had a script somewhere to do part of the job (nothing fancy, only some regular expressions), I'll find it and push it :+1:

Reply:
Sounds good!